MerlinAU.sh

```diff
@@ -1687,8 +1687,7 @@ _CreateEMailContent_()
                Say "${REDct}**ERROR**${NOct}: Unable to send post-update email notification [Saved info is empty]."
                return 1
            fi
-           if [ "$savedNewUpdateVersion" = "$fwInstalledVersion" ] && \
-              [ "$savedInstalledVersion" != "$fwInstalledVersion" ]
+           if [ "$savedNewUpdateVersion" = "$fwInstalledVersion" ]
```

Comment:
Remove the requirement for them to not match, if the installed version matches what we were installing, that is good enough. Reason is offline updates update the settings file now, and allow us to reflash the same version ontop.. With this, if I reinstall the same firmware on top it thinks it failed.

Reply:
> Remove the requirement for them to not match, if the installed version matches what we were installing, that is good enough. Reason is offline updates update the settings file now, and allow us to reflash the same version ontop.. With this, if I reinstall the same firmware on top it thinks it failed.
>
>
Got it. Makes sense.